### DISCUSSION TITLE
ambiguity vs specificity

### DISCUSSION THREAD

**Discussion by axecopfire** [Vets-Who-Code/course-development-guide#6] [General]
How do we handle ambiguity vs specificity in the course material for VWC. If it's overly ambiguous students won't understand what to do. If it's overly specific then there'll be no need for creativity.

**Comment by myquite** [comment]
I've seen learners struggle with ambiguity in standards because instead of viewing them as an opportunity to flex, they are usually more concerned with what the instructor/evaluator wants to see. In the case of loose standards, it isn't clear how the task will be evaluated. 

Being specific about evaluation standards, especially in the early stages of the curriculum, will improve learner outcomes. I view the specificity in standards as constraints designed to encourage the learner to focus on the tasks we want them to demonstrate without adding additional cognitive load. 

These constraints shouldn't negatively impact creativity and, in my view, will encourage a more practical kind of creativity closer to what you could expect on the job.

**Reply by axecopfire** [reply]
Can you outline at what level of specificity you're advocating for here. On a scale of specificity what are you looking for here, from "If member is on Windows machine, then open the cmd program, which can be searched for via the search field [citation] and navigate to...".  to "Open a terminal"

**Reply by myquite** [reply]
Let's be more specific in instances where it makes sense and adds value for the learner. We can leave them more loosely defined as a default and adjust as needed.
